### DISCUSSION TITLE
Add a option to remove "Exit" item

### DISCUSSION THREAD

**Discussion by xmha97** [File-New-Project/EarTrumpet#628] [Ideas]
Add a option to remove "Exit" item
![Taskbar](https://user-images.githubusercontent.com/23485114/106406532-c62d0d00-644e-11eb-95cd-c5036623a1c4.png)

**Comment by davuxcom** [comment]
What problem does removing Exit solve? How would a user exit the program?

**Reply by SmilerRyan** [reply]
Maybe when it starts/quits, it should hide/show the original sound icon for you.

**Comment by xmha97** [comment]
I disabled system volume control:
![Taskbar](https://user-images.githubusercontent.com/23485114/106407330-04c3c700-6451-11eb-95e6-fcc927cffa0b.png)
I don't want close EarTrumpet.
